cmd/kubelet/app/server.go

```diff
@@ -720,6 +720,7 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.Dependencies, stopCh <-chan
 	if err := RunKubelet(s, kubeDeps, s.RunOnce); err != nil {
 		return err
 	}
+	kubeDeps.EventBroadcaster.StartRecordingToSink(stopCh)
```

Comment:
If you add this, you should revisit/remove `makeEventRecorder()` that is called also in this function.